content/docs/errors/faq.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'FAQ: Content API Errors'
+---
+
+## Invalid or undefined branch
+The current branch that Tina is using is invalid or undefined. Double check that the correct branch is selected and it does in fact exist.
+## .tina directory (with generated subdirectory) not pushed to git
+The `.tina` directory and everything inside needs to be pushed up as well. Be sure to add it to your git repository (and make sure you don't have it listed in a `.gitignore`). 
+## API URL is misformatted
+The ContentAPI URL isn't formatted correctly. {Is there a resource we can link to here to provide more info?}
```

Comment:
Maybe https://tina.io/docs/tina-cloud/connecting-site/#enabling-tina-cloud-in-tinacms ?

Reply:
That's exactly what I was hoping for